rand.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2012-2023 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !go1.20
+// +build !go1.20
+
+// A Go client for the NATS messaging system (https://nats.io).
+package nats
+
+import (
+	"math/rand"
+	"time"
+)
+
+func init() {
+	// This is not needed since Go 1.20 because now rand.Seed always happens
+	// by default (uses runtime.fastrand64 instead as source).
+	rand.Seed(time.Now().UnixNano())
```

Comment:
we don't need it now probably but from Go 1.19 it was possible to get call into fastrand64 like this: `maphash.Bytes(maphash.MakeSeed(), nil)`

Reply:
This a new file?

Reply:
yes a new file to add the build tags so that this does not apply to Go +1.20

Reply:
ok so just copyright 2023..